{
  "schema_version": "1.2.0",
  "id": "GHSA-fj35-m94r-9h4c",
  "modified": "2022-07-28T12:14:53Z",
  "published": "2022-05-24T17:17:59Z",
  "aliases": [
    "CVE-2020-12889"
  ],
  "summary": "MISP-maltego incorrectly shares a MISP connection across users",
  "details": "MISP MISP-maltego 1.4.4 incorrectly shares a MISP connection across users in a remote-transform use case.",
  "severity": [

  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "misp-maltego"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1.4.5"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-12889"
    },
    {
      "type": "WEB",
      "url": "https://github.com/MISP/MISP-maltego/commit/3ccde66dab4096ab5663e69f352992cc73e1160b"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/MISP/MISP-maltego"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-284"
    ],
    "severity": "HIGH",
    "github_reviewed": false
  }
}